tests/chainer_tests/functions_tests/pooling_tests/test_unpooling_2d.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+import numpy
+import six
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestUnpooling2D(unittest.TestCase):
+
+    def setUp(self):
+        # Avoid unstability of numerical gradient
```

Comment:
I think forward and backward of unpooling are not unstable, meaning that small perturbation of input causes big change in output.
The reason for unstable-ness of max pooling is that if the largest and second-largest values in one window are close, small changes of these values can change the argmax, which leads huge change in gradient.